pom.xml

```diff
@@ -316,19 +316,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <version>${maven-surefire-plugin.version}</version>
-        <configuration>
-          <argLine>@{argLine}</argLine>
-          <systemPropertyVariables>
-            <log4j.configuration>
-              ${surefire-log4j.file}
-            </log4j.configuration>
-          </systemPropertyVariables>
-        </configuration>
-      </plugin>
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-failsafe-plugin</artifactId>
```

Comment:
does it also make sense to move failsafe to plugin management too?

Reply:
Emmm, I'm not sure about the scope of failsafe plugin. 
Also this pr is try to fix flaky testes and expand timeout time. 
Should we take care of it here or maybe be we can step another PR to deal with failsafe :)